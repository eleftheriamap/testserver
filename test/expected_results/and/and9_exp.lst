
test/exp/test/gen/and/and9_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	d1058a10 	sub	x16, x16, #0x162
   4:	51695d08 	sub	w8, w8, #0xa57, lsl #12
   8:	8a080219 	and	x25, x16, x8
   c:	8a000000 	and	x0, x0, x0
