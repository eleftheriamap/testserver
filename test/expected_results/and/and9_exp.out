Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 00000000ff5a9000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = fffffffffffffe9e
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 00000000ff5a9000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0xd1058a10
0x00000004 : 0x51695d08
0x00000008 : 0x8a080219
0x0000000c : 0x8a000000
